tests/Kiota.Builder.Tests/KiotaBuilderTests.cs

```diff
@@ -6390,12 +6409,17 @@ public void SinglePathParametersAreDeduplicated()
         Assert.NotNull(managerRB);
         var managerUrlTemplate = managerRB.FindChildByName<CodeProperty>("UrlTemplate", false);
         Assert.NotNull(managerUrlTemplate);
-        Assert.Equal("{+baseurl}/users/{id}/manager", managerUrlTemplate.DefaultValue.Trim('"'));
+        Assert.Equal("{+baseurl}/users/{users%2Did}/manager", managerUrlTemplate.DefaultValue.Trim('"'));
```

Comment:
@baywet after updating to kiota 1.9.0 from preview this is causing a regression
in my swagger file i have: `files/{fileid}/dependencies`
 - this generates template: `{+baseurl}/files/{files%2Did}/dependencies`
 - when calling `_api.Files[123].Dependencies.GetAsync()` the url template parameters still has `fileid:123`
 
Eventually `GET api/files//dependencies` is called and the request fails
Any suggestion for this case?

Reply:
Thanks for pointing this. Can you join the conversation at #3901 please?